source/index.js

```diff
@@ -97,3 +97,6 @@ module.exports = request => {
 
 	return timings;
 };
+
+module.exports = timer;
+module.exports.default = timer;
```

Comment:
We don't do this anymore either. See: https://github.com/sindresorhus/mem/issues/31